### ISSUE TITLE
NTR: Blood loss during delivery (Cardiac & vascular)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7788]
New term request Blood loss during delivery (segal_211228163247): Placenta previa; Placental abruption (abruptio placentae), twin-twin transfusion included  UMLS: 0566677

**Comment by pnrobinson** [comment]
@MickeySegal

I think this would need to be quantified in some way so it is clear when the term pertains. Thoughts?

**Comment by MickeySegal** [comment]
I like the definition at https://www.merckmanuals.com/home/women-s-health-issues/complications-of-labor-and-delivery/excessive-uterine-bleeding-at-delivery except that it uses pints:
"Blood loss is considered excessive if one of the following occurs within 24 hours of delivery:
- More than 2 pints of blood are lost.
- A woman has symptoms of significant blood loss, such as low blood pressure, a rapid heart rate, dizziness, light-headedness, fatigue, and weakness."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000325
- label: Excessive blood loss during delivery
- parents: [Past obstetric history (HP:0032467)](https://hpo.jax.org/app/browse/term/HP:0032467)
- pmid: PMID:33913640
- def: Loss of a substantial amount of blood during delivery (some definitions stipulate 1000 ml or more), typically associated with manifestations such as low blood pressure, a rapid heart rate, dizziness, light-headedness, fatigue, and weakness.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7788
- synonyms:
